README.md

```diff
@@ -88,4 +88,18 @@ If you install Vagrant, you can run the application in its own vm. Just install
 ```bash
 $ vagrant up
 
-```
\ No newline at end of file
+## Running in Docker
+If you want to build and run a docker image of this application you can run:
+```bash
+$ docker build --tag todo-app .
+```
+
+to build the application and then to run it run:
+```bash
+$ docker run todo-app -p 5000
```

Comment:
When running the development container, the exercise asks that you do so with a bind mount.

This would mean that you are able to modify the code and see the change without having to rebuild/rerun anything. (`--mount type=...etc`)